Makefile

```diff
@@ -13,6 +14,7 @@ $(DISTFILE):
 	@rm -rf $(DISTDIR)
 	mkdir $(DISTDIR)
 	cp -ar * $(DISTDIR)
+	echo "$(VERSION) ($(GIT_HASH))" > $(DISTDIR)/emscripten-version.txt
```

Comment:
looks like this repurposes `emscripten-version.txt` to also contain the git hash? perhaps a separate file is another option for that, which seems safer for people currently packaging emscripten that might be surprised by this.

Reply:
Oh sorry, I didn't see this comment.  Yes, I am adding to this file.

Reply:
Do you really think that external prcocesses are reading this file.  Isn't parsing `--version` the correct way to determine the emscripten version?   I'm not sure we want external projects depending our internals like that.  But out of an abundance of caution I will put this data in a separate file.

Reply:
Yeah, I agree 100% with each part of that. Just seems like an abundance of caution is reasonable with things like this.